frontend/src/metabase/search/components/filters/SQLNativeQueryFilter/SQLNativeQueryFilter.tsx

```diff
@@ -0,0 +1,9 @@
+import { t } from "ttag";
+import type { SearchFilterToggle } from "metabase/search/types";
+
+export const SQLNativeQueryFilter: SearchFilterToggle = {
+  label: t`Search the contents of SQL queries`,
```

Comment:
This should read "Search the contents of native queries", because some DBs don't use SQL